files/image_config/monit/monitoring_containers

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+
+import swsssdk
+
```

Comment:
Delete this blank line, grouping swsssdk and sonic_py_common together, as both are third-party libraries in relation to this script.

Reply:
Great suggestions! Updated.